.github/workflows/cr.yml

```diff
@@ -0,0 +1,36 @@
+name: cr
+on:
+  push:
+    branches: [main]
+    tags: ['!**'] # Avoid publishing on tags
+  pull_request:
+    types: [opened, synchronize, labeled] # Run on PR creation, updates, and when labels are added
```

Comment:
Run the workflow when pull requests are opened/updated, or when a label is added

I used https://github.com/vitest-dev/vitest/blob/main/.github/workflows/cr.yml for reference